biooutput.py

```diff
@@ -0,0 +1,101 @@
+import pandas as pd
+import numpy as np
+
+class Biooutput():
+    """Output Class
+
+    """
+
+    def __init__(self, hr_data3):
+        self.data = hr_data3
+
+    def summarize_ecg(self):
+        """Create txt file summarizing ECG analysis
+
+        :param data: pandas dataframe {'Heartrate', 'B\T', 'time'}
+        :return ecgResults.txt
+        """
+
+        # Finds the last time point in the data file
+        maxTime = self.data['time'].iat[-1]
+
+        # Tells user the purpose of this program
+        printStr = ("This program will summarize the ECG data.\n\
+        Data recorded from 0s to {}s.\n\
+        Enter a start and stop time in that range to get an instantaneous \n\
+        heart rate at those times, an average rate over that period, \n\
+        and any bradycardia or tachycardia events between those times.\n\n".format(maxTime))
+        print(printStr)
+
+        # Error messages for possible errors
+        errmsg_max = ('\nThe time you selected is longer than the recorded data!\n\
+        Please choose between 0s and {}s\n\n'.format(maxTime))
+        errmsg_notint = ('\nNot an integer! Please choose a time between 0s and {}s\n\n'.format(maxTime))
+
+        # Get user specified period for averaging heartrate
+        startTimestr = 'Start time (in seconds): '
+
+        # Check for valid input
+        while True:
+            try:
+                startTime = int(input(startTimestr))
+            except ValueError:
+                print(errmsg_notint)
+            else:
+                if startTime > maxTime:
+                    print(errmsg_max)
+                else:
+                    startTime = int(startTime)
+                    break
+
+        # Get user specified stop time
+        stopTimestr = 'Stop time (in seconds): '
+
+        # Check for valid input
+        while True:
+            try:
+                stopTime = int(input(stopTimestr))
+            except ValueError:
+                print(errmsg_notint)
+            else:
+                break
+
+        # If user gives time later than maxTime just set stopTime to maxTime
+        if stopTime > maxTime:
+            stopTime = maxTime
+
+        # Tell user what is happening
+        printStr = "ECG data will be summarized from {}s to {}s'".format(startTime, stopTime)
+        print(printStr)
+
+        # Convert user input into indices - data is broken down in the dataframe file in chunks of 5seconds
+        startInd = int(np.floor(startTime / 5))
```

Comment:
snake_case

Reply:
Fixed. Changed variables that needed to be snake_case in biooutput.py in the latest commit to develop_io branch.